### PR TITLE
🐛 Fix teleport + access

### PR DESCRIPTION
**Pull request by bakayolo**
- only forward service `CATALOG` or `BACKEND_API`
- remove 2 useless `access` on root cmd
- fix teleport -> will teleport 1 service running locally on port 8080
- get latest protos from kkc
- fix GetLatestSuccessfulRelease to only get ROLLBACK and DEPLOY release

### CODE REVIEW

File: internal/cli/cli.go [hunk 1]
```diff
@@ -78,8 +76,8 @@ func (c *Cli) Execute(controller controller.ControllerInterface) {
 func createInitCommand(controller controller.ControllerInterface) *cobra.Command {
 	initCmd := &cobra.Command{
 		Use:   "init",
-		Short: "Login to an existing KintoHub account",
-		Long:  `Helps you to log in an already existing KintoHub account`,
+		Short: "Init the cli",
+		Long:  `Create a 'kinto.yaml' file in your home directory`,
```
Comment (sumitkolhe) [inline internal/cli/cli.go:80]: `init` does not create a kinto.yaml file. as of now it is just for setting/resetting the master host

**Review by sumitkolhe — APPROVED** [review]
LGTM

File: internal/utils/helper.go [hunk 2]
```diff
@@ -107,28 +109,36 @@ func CompareGitUrl(remoteGitUrl string) bool {
...
 		if err != nil {
 			TerminateWithError(err)
 		}
 		return port
 	}
 }
 
+func CanPortForwardToRelease(release *types.Release) bool {
+	if release.RunConfig != nil &&
+		(release.RunConfig.Type == types.Block_BACKEND_API || release.RunConfig.Type == types.Block_CATALOG) {
```
Comment (sumitkolhe) [inline internal/utils/helper.go:135]: shouldn't we be allowing forwarding for web-apps also?

Comment (bakayolo) [reply internal/utils/helper.go:135]: I am not sure, why would anyone make a website private? we could consider it if someone ask for it but I don't see the use case

**Pull request merged**